client/components/google/google.maps.directives.js

```diff
@@ -371,6 +372,14 @@ app.directive('googleMarker', function ($timeout,$compile) {
             };
           }
 
+          function overrideCloseBounds() {
```

Comment:
Nice work ;) 

My only comment would be to move this above the function that calls it.

After that, we should be good to go.